docker/docker-compose.yml

```diff
@@ -74,6 +74,8 @@ services:
     environment:
       MONGO_INITDB_ROOT_USERNAME: root
       MONGO_INITDB_ROOT_PASSWORD: example
+    volumes:
+      - ./mongo-data:/data/db
```

Comment:
👌  remember to add this to the gitignore since it's now a folder created inside the repo (same as done for `docker/minio-data` or `docker/dynamo-db-data` for example)